tools/cross/format.py

```diff
@@ -189,18 +217,18 @@ class FormatConfig:
     # FormatConfig(
     #     directory="src/workerd", globs=("*.c++", "*.h"), formatter=clang_format
     # ),
-    # FormatConfig(
-    #     directory="src",
-    #     globs=("*.js", "*.ts", "*.cjs", "*.ejs", "*.mjs"),
-    #     formatter=prettier,
-    # ),
-    # FormatConfig(directory="src", globs=("*.json",), formatter=prettier),
+    FormatConfig(
```

Comment:
was prettier intended to be included? Imo it requires running `npm install` or something and is not fetched automatically?

Reply:
It's listed as a runtime dependency of workerd, so it's somewhere... I'm currently doing a clean build to find out where it puts it.

Reply:
Found it (bazel-bin/node_modules/prettier/bin/prettier.cjs)